examples/timer32.rs

```diff
@@ -56,119 +61,112 @@ fn main() -> ! {
     let mut p2_1 = gpio.p2_1.into_output();
     let mut p2_2 = gpio.p2_2.into_output();
 
-    let mut tim32 = p.TIMER32.constrain().set_clock(_clock);
+    let tim32 = p.TIMER32.split().set_clock(_clock);
+    let mut timer0 = tim32.channel0;
+    let mut timer1 = tim32.channel1;
+    let tim32 = tim32.tim.unwrap();
```

Comment:
Is not a good practice to redefine a var like this

Reply:
Ok